src/logger.cpp

```diff
@@ -0,0 +1,87 @@
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <locale>
+#include <sstream>
+
+#include "logger.h"
```

Comment:
The very first include must always be the precompiled header `#include <pch.h>`, otherwise you get no benefit from it. Library headers should always be included through it, only local headers should be directly included.

Applies to the header too (after the `#pragma once`).